webextension/css/popup.css

```diff
@@ -1,16 +1,55 @@
 /* General Rules and Resets */
-body {
-  inline-size: 300px;
-  max-inline-size: 300px;
+* {
+  font-size: inherit;
+  margin-block-end: 0;
+  margin-block-start: 0;
+  margin-inline-end: 0;
+  margin-inline-start: 0;
+  padding-block-end: 0;
+  padding-block-start: 0;
+  padding-inline-end: 0;
+  padding-inline-start: 0;
```

Comment:
I think this is actually a product of the text-size actually. Everything else looks correct, it might be a dpi change for the text-size needed.

Some of those middle container rows are still hard coded so a little more fragile for the text size and layout.